.github/dependabot.yml

```diff
@@ -76,10 +76,12 @@ updates:
           - "com.icegreen.greenmail*"
     ignore:
       - dependency-name: "*"
-        update-types: ["version-update:semver-major"]
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
       - dependency-name: "de.bund.bva.isyfact:*"
 
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
\ No newline at end of file
+      interval: "weekly"
+    commit-message:
+      prefix: "ci"
```

Comment:
Braucht man hier noch einen Doppelpunkt?

Reply:
Der Doppelpunkt wird automatisch hinzugefügt https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message